sarpy/io/complex/sicd_elements/base.py

```diff
@@ -162,7 +163,7 @@ def parse_string(val):
         return None
     if isinstance(value, bool):
         return value
-    elif isinstance(value, integer_types):
+    elif isinstance(value, (integer_types, numpy.bool_)):
```

Comment:
integer_types is potentially already a tuple, but I'll fix this

Reply:
Good catch, properly constructing a combined tuple is the way to go.

Reply:
I think  it's easier to just do isinstance(-, integer_types) or isinstance(-, numpy.bool), since integer_types may also NOT be a tuple. I'm half-ass cheating into 32/64 bit 2.7 and 3 compliance. I totally hate having to do this.